client/src/index.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import { BrowserRouter as Router} from 'react-router-dom';
-
-
-ReactDOM.render(
-  <Router>
-    <App />
-  </Router>,
-  document.getElementById('root')
-);
-
+import React from 'react'
+import ReactDOM from 'react-dom'
+import App from './App'
+import { BrowserRouter as Router } from 'react-router-dom'
+import { Context } from "./components/DataManager"
+
+
+ReactDOM.render(
+    <Router>
+        <Context.Consumer>

```

Comment:
Would need to be <Context.Provider> 

(If selected to call it this from previous change above).... Or, would be Provider/DataManager